libs/Functions.php

```diff
@@ -511,16 +511,24 @@ public function post_author($options=array()){
 	}
 
 	//Post type meta
-	public function post_type($icon=true){
+	public function post_type($options=array() ){
+		$defaults = apply_filters('nebula_post_type_defaults', array(
+			'icon' 		=> true, // boolean true for generic defaults, boolean false to disable icon, or string of class name(s) for icon.
```

Comment:
For consistency with the rest of Nebula, we can remove the tabs for the array definitions.